### DISCUSSION TITLE
What to use to open on amd graphics cards?

### DISCUSSION THREAD

**Discussion by ByBluex** [hacksider/Deep-Live-Cam#351] [Q&A] [answered]
what to use to open on amd graphics cards? İ use rx 6600xt

**Comment by Syntoxr** [comment, marked as answer]
Use directML like described [here](https://github.com/hacksider/Deep-Live-Cam?tab=readme-ov-file#directml-execution-provider-windows)

Also use `python run.py --execution-provider dml` (`dml `instead of `directml`) in latest version
